Bottom: fb340b6752f4783c69c737f22f8364579cb78b57
Top:    f2134d7e1dc3f7f00b7b049ee7a86f04444091c1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-07 23:23:43 +0100

Refresh of crasy-test

---

diff --git a/src/draw.c b/src/draw.c
index 99f20a9..b5dafe3 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -228,99 +228,6 @@ EMark_callback (const BoxType * b, void *cl)
 }
 
 static int
-<<<<<<< current
-hole_callback (const BoxType * b, void *cl)
-{
-  PinTypePtr pv = (PinTypePtr) b;
-  int plated = cl ? *(int *) cl : -1;
-
-  if ((plated == 0 && !TEST_FLAG (HOLEFLAG, pv)) ||
-      (plated == 1 &&  TEST_FLAG (HOLEFLAG, pv)))
-    return 1;
-
-  if (TEST_FLAG (THINDRAWFLAG, PCB))
-    {
-      if (!TEST_FLAG (HOLEFLAG, pv))
-        {
-          gui->set_line_cap (Output.fgGC, Round_Cap);
-          gui->set_line_width (Output.fgGC, 0);
-          gui->draw_arc (Output.fgGC,
-                         pv->X, pv->Y, pv->DrillingHole / 2,
-                         pv->DrillingHole / 2, 0, 360);
-        }
-    }
-  else
-    gui->fill_circle (Output.bgGC, pv->X, pv->Y, pv->DrillingHole / 2);
-
-  if (TEST_FLAG (HOLEFLAG, pv))
-    {
-      if (TEST_FLAG (WARNFLAG, pv))
-        gui->set_color (Output.fgGC, PCB->WarnColor);
-      else if (TEST_FLAG (SELECTEDFLAG, pv))
-        gui->set_color (Output.fgGC, PCB->ViaSelectedColor);
-      else
-        gui->set_color (Output.fgGC, Settings.BlackColor);
-
-      gui->set_line_cap (Output.fgGC, Round_Cap);
-      gui->set_line_width (Output.fgGC, 0);
-      gui->draw_arc (Output.fgGC,
-                     pv->X, pv->Y, pv->DrillingHole / 2,
-                     pv->DrillingHole / 2, 0, 360);
-    }
-  return 1;
-}
-
-void
-DrawHoles (bool draw_plated, bool draw_unplated, const BoxType *drawn_area)
-{
-  int plated = -1;
-
-  if ( draw_plated && !draw_unplated) plated = 1;
-  if (!draw_plated &&  draw_unplated) plated = 0;
-
-  r_search (PCB->Data->pin_tree, drawn_area, NULL, hole_callback, &plated);
-  r_search (PCB->Data->via_tree, drawn_area, NULL, hole_callback, &plated);
-}
-
-static void
-_draw_line (LineType *line)
-{
-  gui->set_line_cap (Output.fgGC, Trace_Cap);
-  if (TEST_FLAG (THINDRAWFLAG, PCB))
-    gui->set_line_width (Output.fgGC, 0);
-  else
-    gui->set_line_width (Output.fgGC, line->Thickness);
-
-  gui->draw_line (Output.fgGC,
-		  line->Point1.X, line->Point1.Y,
-		  line->Point2.X, line->Point2.Y);
-}
-
-static void
-draw_line (LayerType *layer, LineType *line)
-{
-  if (TEST_FLAG (SELECTEDFLAG | FOUNDFLAG, line))
-    {
-      if (TEST_FLAG (SELECTEDFLAG, line))
-        gui->set_color (Output.fgGC, layer->SelectedColor);
-      else
-        gui->set_color (Output.fgGC, PCB->ConnectedColor);
-    }
-  else
-    gui->set_color (Output.fgGC, layer->Color);
-  _draw_line (line);
-}
-
-static int
-line_callback (const BoxType * b, void *cl)
-{
-  draw_line ((LayerType *) cl, (LineType *) b);
-  return 1;
-}
-
-static int
-=======
->>>>>>> patched
 rat_callback (const BoxType * b, void *cl)
 {
   RatType *rat = (RatType *)b;
